src/seedu/addressbook/AddressBook.java

```diff
@@ -208,7 +208,20 @@
 
     public static void main(String[] args) {
         showWelcomeMessage();
-        processProgramArgs(args);
+        //processProgramArgs(args);
+        if (args.length >= 2) {
+            showToUser(MESSAGE_INVALID_PROGRAM_ARGS);
```

Comment:
commit message should be in imperative tone (i.e. ‘add’ rather than ‘adding’ or ‘added’) and meaningful